housing_cooperative_base/models/res_partner.py

```diff
@@ -32,6 +32,10 @@ class ResPartner(models.Model):
         comodel_name='res.country',
         string='Citizenship',
         ondelete='restrict')
+    study = fields.Many2one(
+        comodel_name='hc.studies',
+        string='Study',
+        ondelete='restrict')
```

Comment:
Never took the time to understand, maybe you can clarify  the three options (set null, cascade, restrict) for me:

- Which is default?
- What happens if I delete a study object?
- What happens if i delete a partner object?